ci/check-code-format.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -ex
+
+set +e
```

Comment:
Is there a reason to call both `set +e` and `set -ex`? Seems like this might be equivalent to `set -x` (assuming set +e is default behavior)